apps/frontend/src/pages/access-management/libs/components/group-users-table/group-users-table.tsx

```diff
@@ -0,0 +1,100 @@
+import { type FieldErrors, type UseFormSetValue } from "react-hook-form";
+
+import {
+	Loader,
+	Table,
+	TablePagination,
+} from "~/libs/components/components.js";
+import { EMPTY_LENGTH } from "~/libs/constants/constants.js";
+import { DataStatus } from "~/libs/enums/enums.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useEffect,
+	usePagination,
+} from "~/libs/hooks/hooks.js";
+import {
+	actions as groupActions,
+	type GroupCreateRequestDto,
+} from "~/modules/groups/groups.js";
+
+import { getUserColumns, getUserRows } from "../../helpers/helpers.js";
+import { type UserRow } from "../../types/types.js";
+import styles from "./styles.module.css";
+
+const getRowId = (row: UserRow): number => row.id;
```

Comment:
the logic for it will always be really simple, so let's define it inside the component itself, without a separate function

```
getRowId={(row): number => row.id}
```

Reply:
then let's just keep it inside the component